backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -14,12 +15,16 @@ import { UserAdmin, SearchUsersResult } from '@model/UserAdmin'
 import { PendingCreation } from '@model/PendingCreation'
 import { CreatePendingCreations } from '@model/CreatePendingCreations'
 import { UpdatePendingCreation } from '@model/UpdatePendingCreation'
+import { ContributionLink } from '@model/ContributionLink'
+import { ContributionLinkList } from '@model/ContributionLinkList'
 import { RIGHTS } from '@/auth/RIGHTS'
 import { UserRepository } from '@repository/User'
 import CreatePendingCreationArgs from '@arg/CreatePendingCreationArgs'
 import UpdatePendingCreationArgs from '@arg/UpdatePendingCreationArgs'
 import SearchUsersArgs from '@arg/SearchUsersArgs'
+import ContributionLinkArgs from '@arg/ContributionLinkArgs'
 import { Transaction as DbTransaction } from '@entity/Transaction'
+import { ContributionLink as DbContributionLink } from '@entity/ContributionLink'
```

Comment:
the entity should have quite the same name as the dbtable, because the entity mirrors the database. The TypeORM-Model will then distigue between a single dataobject or container/list/array with several dataobjects.
examples:

    entity: Users <-> table: users <-> typeORM(s): User
    entity: Transactions <-> table: transactions <-> typeORM(s): Transaction
    entity: TransactionLinks <-> table: transaction_links <-> typeORM(s): TransactionLink / TransactionLinkList
    entity: ContributionLinks <->table: contribution_links <-> typeORM(s): ContributionLink / ContributionLinkList

so please rename the entity ContributionLink to ContributionLinks